packages/api-client/src/components/ImportCollection/hooks/useUrlPrefetcher.ts

```diff
@@ -12,34 +18,96 @@ export type PrefetchResult = {
  * Core logic for fetching and processing a URL
  */
 export function createUrlPrefetcher() {
```

Comment:
I'm not sure I understand why this is a factory? Can't we just use prefetchUrl directly?

Reply:
maybe I'm missing something 😕

Reply:
True! It was necessary before, but not anymore. Let’s keep it simple. 👍